[media-library][2/2] Make moving assets unnecessary - Add `initialAssetLocalUri` parameter to `createAlbumAsync` (#35692)
expo/expo: docs/pages/versions/unversioned/sdk/media-library.mdx

</SnackInline>

> ## Android Limitations

Reviewer: ```suggestion
## Known limitations
```

Let's use the same pattern we did/currently do for the API references.

> ### Empty Albums

Reviewer: ```suggestion
### Empty albums
```

Sentence case instead title case for section titles.

> Due to system limitations, it is not possible to create empty albums on Android. It is necessary to either pass an existing asset to add to the album or a URI of a local resource, which will be used to create a new asset inside the album.

Reviewer: ```suggestion
Due to system limitations on Android, it is not possible to create empty albums. It is necessary to either pass an existing asset to add to the album or a URI of a local resource, which will be used to create a new asset inside the album.
```

> ### Moving Assets Between Albums

Reviewer: ```suggestion
### Moving assets between albums
```

Android 11 introduced permission changes that make the operation of moving assets between albums require confirmation from the user every time.

> Therefore, when creating a new asset, instead of creating the asset and then moving it to the album, it is recommended to pass the `album` parameter to the [`createAssetAsync`](#medialibrarycreateassetasynclocaluri-album) method,
> which will automatically add the asset to the album without the need for user confirmation.

Reviewer: ```suggestion

Therefore, when creating a new asset, instead of creating the asset and then moving it to the album, it is recommended to pass the `album` parameter to the [`createAssetAsync`](#medialibrarycreateassetasynclocaluri-album) method, which will automatically add the asset to the album without the need for user confirmation.
```

Let's improve the readability here by making this a second paragraph.

</SnackInline>

## Known limitations

### Empty albums

> Due to system limitations on Android, it is not possible to create empty albums. It is necessary to either pass an existing asset to add to the album or a URI of a local resource, which will be used to create a new asset inside the album.

Reviewer: ```suggestion
Due to system limitations on Android, it is impossible to create empty albums. It is necessary to either pass an existing asset to add to the album or a URI of a local resource, which will be used to create a new asset inside the album.
```

Sorry I missed this in my previous review. Let's fix the Vale warning about wordiness.
